workflow-job

```diff
@@ -466,6 +476,26 @@ flows["${sha1}"] = {
             dockeringGuiceCassandraLdap.success()
         }
 
+        stage "Deploying Guice Server with JPA then provision data ${buildId}"
+        node {
+            sh "cd dockerfiles/run/guice/provisioned; cp Dockerfile NewDockerfile; sed -i -- 's,linagora/james-jpa-guice,${images.jamesJPA},g' NewDockerfile"
+
+            sh "cd dockerfiles/run/guice/provisioned; docker build --tag=${images.jamesJpaSampleForTesting} -f NewDockerfile ."
+
+            dockeringGuiceJPASampleTesting.success()
+
+            sh "docker run --detach=true --name=${containers.jamesJpaSampleForTesting} --hostname ${hostname} --expose=143 --publish-all=true ${images.jamesJpaSampleForTesting}"
```

Comment:
I'm not expert on docker. I've just more read on that and you are right. But this one starts a container just for testing so the way it expose redundant port for testing and don't care about the dockerfile. I think we accept that redundant.